app/graph/types/user_type.rb

```diff
@@ -17,4 +17,18 @@ class UserType < DefaultObject
   def source
     Source.find(object.source_id)
   end
+
+  def name
+    meedan_user? ? 'Meedan' : object.name
```

Comment:
This hard-coded "Meedan" should some from a new configuration key... something like `CheckConfig.get('super_user_name')`. Then in `config/config.yml.example` you can set it as "Meedan".